github-metrics/get-github-metrics.js

```diff
@@ -0,0 +1,123 @@
+import { Octokit } from "octokit";
+
+class ReferrerTraffic {
+    constructor(referrer, count, uniques) {
+        this.referrer = referrer;
+        this.count = count;
+        this.uniques = uniques;
+    }
+}
+
+class TopPaths {
+    constructor(path, count, uniques) {
+        this.path = path;
+        this.count = count;
+        this.uniques = uniques;
+    }
+}
+
+async function getGitHubMetrics(owner, repo) {
```

Comment:
```suggestion
// @todo set up a server to run this job regularly since we can't specify the date range

async function getGitHubMetrics(owner, repo) {
```

Reply:
I don't think I'm planning to set up a server to do this any time soon. I'm just going to set a reminder in my calendar to do it manually. Otherwise, we'd need to set up an EC2 instance or something to be "always on", and some sort of monitoring to alert us if the server goes down/the job doesn't run, and that seems to add unnecessary complexity at the moment to an internal tool we're using for one project.

If we get to a point where we're using it for a lot of projects or it becomes mission critical, this may be something we want to do in the future, but it doesn't seem worthwhile to do it now.